staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -263,10 +266,11 @@ type Cacher struct {
 	// indexedTrigger is used for optimizing amount of watchers that needs to process
 	// an incoming event.
 	indexedTrigger *indexedTriggerFunc
-	// watchers is mapping from the value of trigger function that a
-	// watcher is interested into the watchers
+	
 	watcherIdx int
-	watchers   indexedWatchers
+	// watchers is mapping from both watcherIdx and value of trigger function that a watcher is interested in to the watchers.
+	// Else, mapping exists between watcherIdx and the watcher that the index identifies.
```

Comment:
I don't understand it. Let's remove and say above that trigger just may be empty.

Reply:
The earlier comment didn't mention of 2 mapping of the `watchers` field, namely `watcherIdx` and the trigger function, so I updated this. Should I just remove the entire thing to "Trigger just may be empty" ?